handbook/product/README.md

```diff
@@ -127,7 +127,7 @@ Once outcomes have been approved by the CEO, Product Operations will complete th
 Product Managers [write user stories](https://fleetdm.com/handbook/company/development-groups#writing-a-good-user-story) in the [drafting board](https://app.zenhub.com/workspaces/-product-backlog-coming-soon-6192dd66ea2562000faea25c/board). The drafting board is shared by every [product group](https://fleetdm.com/handbook/company/development-groups).
 
 ## Drafting user stories
-Product Designers [draft user stories](https://fleetdm.com/handbook/company/development-groups#drafting).
+Product Designers [draft user stories](https://fleetdm.com/handbook/company/development-groups#drafting) that have been prioritized by PMs. If the estimated user stories for a product group exceed [that group's capacity](https://fleetdm.com/handbook/company/product-groups), all new design work for that group is paused, and the designer will contribute in other ways (documentation & handbook work, Figma maintenance, QA, etc.) until the PM deprioritizes estimated stories to make room, or until the next sprint begins.
```

Comment:
I didn't quite follow this. How does this affect our current process? Currently, we drop all estimated items that were not pulled into the sprint, so if a designer stops working, will that cause us to have too few items for the next sprint?

FYI @ireedy